CHANGELOG.md

```diff
@@ -0,0 +1,10 @@
+## Unreleased
+
+### Added
+- Migrate API translation code to plugin infra
+- Support for self transpilation with improved error reporting
+- New target: python. Transpiles untyped python to typed python.
+- Directory mode. Cross module type inference.
```

Comment:
"Directory mode" previously existed.

Is there a test which does "Cross module type inference." ?  If they exist, there are not many of them, so I'd be wanting to create a few more tests to verify it works reasonably well.

Reply:
Opened #391